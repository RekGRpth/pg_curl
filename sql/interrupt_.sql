\unset ECHO
\set QUIET 1
\pset format unaligned
\pset tuples_only true
\pset pager off
\set ON_ERROR_ROLLBACK 1
BEGIN;
CREATE EXTENSION pg_curl;
select curl_easy_setopt_url('https://httpbin.org/delay/10', conname:='1');
select curl_easy_setopt_url('https://httpbin.org/delay/7', conname:='2');
set statement_timeout = '5s';
select curl_multi_perform();
ROLLBACK;
