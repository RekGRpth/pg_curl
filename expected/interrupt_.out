\unset ECHO
t
t
ERROR:  Operation was aborted by an application callback
DETAIL:  Callback aborted
